client/src/app/core/repositories/assignments/assignment-repository.service.ts

```diff
@@ -106,4 +109,13 @@ export class AssignmentRepositoryService extends BaseIsAgendaItemAndListOfSpeake
         // await this.httpService.post(restPath, data);
         throw new Error('TODO');
     }
+
+    protected getPayloadForCreateAction(
```

Comment:
I get 5 matches when searching for `getPayloadForCreateAction`, but they never get called (same as `getPayloadForUpdateAction`). I guess they can be removed.